### PR TITLE
Typos from marvin.cs.uidaho.edu: R

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR https://github.com/codespell-project/codespell/pull/1967. I've added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -29783,6 +29835,9 @@ rectiinear->rectilinear
 recude->reduce
 recuiting->recruiting
 reculrively->recursively
+recun->reckon
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29838]: recon, recur?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -29783,6 +29835,9 @@ rectiinear->rectilinear
...
+recund->reckoned
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29839]: `fecund`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -29783,6 +29835,9 @@ rectiinear->rectilinear
...
+recuning->reckoning
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29840]: `retuning`, `recusing`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -29882,6 +29940,8 @@ reencode->re-encode
 reenfoce->reinforce
 reenfoced->reinforced
 reenforced->reinforced
+reenforcement->reinforcement
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29943]: `reenforce->reinforce`
`reenforces->reinforces`
`reenforcing->reinforcing`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -29882,6 +29940,8 @@ reencode->re-encode
 reenfoce->reinforce
 reenfoced->reinforced
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29941]: `reenfoces->reinforces`
`reenfocement->reinforcement`
`reenfocing->reinforcing`

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -30255,6 +30315,10 @@ releae->release
 releaed->released
 releaeing->releasing
 releaes->releases, release,
+releaf->relief
+releafed->relieved
+releafes->relieves
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30320]: `releafs->relieves`?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -30342,6 +30424,11 @@ remaing->remaining
 remainging->remaining
 remainig->remaining
 remainst->remains
+remander->remainder
+remane->remain
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30428]: `rename`, `remake`?

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -30657,11 +30777,18 @@ renforced->reinforced
...
+renig->renege
+reniged->reneged
+reniger->reneger
+reniging->reneging
 rennaisance->renaissance
 rennovate->renovate
 rennovated->renovated
 rennovating->renovating
 rennovation->renovation
+renosance->renaissance
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30789]: `resonance`?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -30657,11 +30777,18 @@ renforced->reinforced
...
+renoun->renown
+renouned->renowned
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30791]: `renounced`?

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -30811,6 +30955,7 @@ reponses->responses
 reponsibilities->responsibilities
 reponsibility->responsibility
 reponsible->responsible
+repore->report, rapport,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30958]: `repose` too?

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -31637,12 +31783,23 @@ rgister->register
...
 rhythimcally->rhythmically
 rhytmic->rhythmic
+rickoshay->ricochet
+rickoshayed->ricocheted
+rickoshaying->ricocheting
+rickoshays->ricochets
 ridiculus->ridiculous
+riendeer->reindeer
+riendeers->reindeers
+rige->ridge
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31801]: `rice`, `ride`, `rigs`

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -31637,12 +31783,23 @@ rgister->register
...
+riges->ridges
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31802]: `rides`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:29838]: Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:29839]: Added.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:29840]: Added.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:29943]: Added.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:29941]: Added.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:30320]: Added.

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:30428]: Added.

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:30789]: Added.

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:30791]: Added.

File: codespell_lib/data/dictionary.txt (hunk 10 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:30958]: Added.

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:31801]: Added.

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:31802]: Added.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**